dkim_verify logerror for a "normal" response

### Haraka version

haraka/Haraka HEAD (as of Oct 4)

### Expected behavior

Emit errors when there's an actual error.

### Observed behavior

````
Oct  4 19:11:19 haraka haraka[67576]: [ERROR] [84638BC5-4BF6-4CE6-9E4A-260DE27E435D.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:11:22 haraka haraka[67576]: [ERROR] [57D4C896-0BC4-4A08-B710-08E6B889D9DD.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:11:42 haraka haraka[67576]: [ERROR] [F2E42F1E-D7AD-4411-B8F4-E81A8E750F56.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:11:53 haraka haraka[67576]: [ERROR] [BD54342B-C6BC-4063-AE5B-DF749B77E2EF.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:12:24 haraka haraka[67576]: [ERROR] [5318D3CB-A9FA-4165-955B-9140EDA43D91.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:12:40 haraka haraka[67576]: [ERROR] [C29FA802-AC76-40F2-A52D-68DF9E24643B.1] [dkim_verify] No results from DKIMVerifyStream
Oct  4 19:13:04 haraka haraka[67576]: [ERROR] [B17BCF12-2973-45F7-9805-E451C66CBD53.1] [dkim_verify] No results from DKIMVerifyStream
````

### Steps to reproduce

Install Haraka. Watch an email sail by without a DKIM signature. See Haraka emit an error.

This is really for @smfreegard 

I think [this line](https://github.com/haraka/Haraka/blob/master/plugins/dkim_verify.js#L23) should be a loginfo. Agree?

--- a/dkim.js
+++ b/dkim.js
@@ -508,7 +508,7 @@ DKIMVerifyStream.prototype.handle_buf = function (buf) {
                 }
                 if (!this.header_idx['dkim-signature']) {
                     this._no_signatures_found = true;
-                    return this.cb(null, this.result);
+                    return this.cb(null, this.result, this.results);
                 }
                 else {
                     // Create new DKIM objects for each header
@@ -555,12 +555,13 @@ DKIMVerifyStream.prototype.write = function(buf) {
 };
 
 DKIMVerifyStream.prototype.end = function(buf) {
-    if (buf) this.handle_buf(buf);
-    else this.handle_buf(null);
+    this.handle_buf(((buf) ? buf : null));
     for (var d=0; d<this.dkim_objects.length; d++) {
         this.dkim_objects[d].end();
     }
-    if (this.pending === 0 && this._no_signatures_found === false) this.cb(null, this.result);
+    if (this.pending === 0 && this._no_signatures_found === false) {
+        this.cb(null, this.result, this.results);
+    }
 };
 
 exports.DKIMVerifyStream = DKIMVerifyStream;

